src/components/chat/comment-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import { BaseComponent } from "../base";
+import DocumentCommentIcon from "../../assets/document-id.svg";
+import SendIcon from "../../assets/send-icon.svg";
+import "./comment-card.scss";
+import "../themes.scss";
+
+interface IProps {
+}
+
+interface IState {
+  commentTextAreaHeight: number | string;
+  commentAdded: boolean;
+  commentText: string;
+}
+
+@inject("stores")
+@observer
+export class CommentCard extends BaseComponent<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+    this.handleCommentTextAreaChange = this.handleCommentTextAreaChange.bind(this);
+    this.handleCancelPost = this.handleCancelPost.bind(this);
+    this.handleSendPost = this.handleSendPost.bind(this);
+
+    this.state = {
+      commentTextAreaHeight: 35,
+      commentAdded: false,
+      commentText: "",
+    };
+  }
+
+  public render() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card ${ui.activeNavTab}`} data-testid="comment-card">
+        {this.renderThreadHeader()}
+        {this.renderCommentTextbox()}
+      </div>
+    );
+  }
+
+  private renderThreadHeader() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card-header ${ui.activeNavTab}`}>
+        <DocumentCommentIcon className="new-thread-header-icon"/>
+      </div>
+    );
+  }
+  handleCommentTextAreaChange = (event: React.FormEvent<HTMLTextAreaElement>) => {
+    const target = event.currentTarget as HTMLTextAreaElement;
+    const targetText = target.value;
+    if (targetText === "") {
```

Comment:
This is a nit and opinions vary so feel free to ignore me if you're not convinced, but I would write this as
```suggestion
    if (!targetText) {
```
because
- it reads (to me) as a simpler test.
- it's a broader test because it handles `null` and `undefined` as well so you don't have to wonder whether there are any circumstances under which a `<textarea>` might return such values, and if so how they should be handled.

The general principle is that (to me) simpler, broader tests leave less room for the reader to wonder whether this particular definition of "empty" is special or whether other possible "empty" values would be handled the same or not.